tests/bearlib/aspects/BaseTest.py

```diff
@@ -1,7 +1,7 @@
 from coalib.bearlib.aspects import aspectclass
 from coalib.bearlib.aspects.base import aspectbase
 
-import pytest
+import unittest
 
 
 class AspectBaseTest:
```

Comment:
I believe @RJ722 tried that in a previous iteration of this patch, and it caused strange errors.
This area of the code has an extreme level of meta-programming, which may be part of the problem.